app/controllers/medicaid/insurance_current_type_controller.rb

```diff
@@ -9,7 +9,7 @@ def skip?
     end
 
     def not_insured?
-      !current_application&.insured?
+      !current_application&.anyone_is_insured?
```

Comment:
What do you think about offloading a liiiittle bit of cognitive friction here and making a `#no_one_is_insured?` method on current application. Unpacking "not anyone is insured" makes my feeble brain pause for longer than it should.

Reply:
agree! was actually just struggling with this when continuing to develop the feature